### PR TITLE
IGAPP-37: Social media preview

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/src/modules/common/containers/Helmet.js [hunk 1]
```diff
@@ -27,12 +28,20 @@ class Helmet extends React.PureComponent<PropsType> {
...
     const { pageTitle, cityModel, metaDescription } = this.props
+    const previewImageUrl = new URL(buildConfig().shareBaseUrl)
+    previewImageUrl.pathname = buildConfig().icons.socialMediaPreview
 
     return <ReactHelmet>
       {pageTitle && <title>{pageTitle}</title>}
       {cityModel && !cityModel.live && <meta name='robots' content='noindex' />}
       {metaDescription && <meta name='description' content={metaDescription} />}
       {this.getLanguageLinks()}
+      {/* https://developers.facebook.com/docs/sharing/webmasters */}
```
Comment (SteffiStuffel) [inline web/src/modules/common/containers/Helmet.js:39]: Is this comment helpful / necessary ?

Comment (maxammann) [reply web/src/modules/common/containers/Helmet.js:39]: I tihnk it is good as it shows how the og: tags work.

**Review by maxammann — APPROVED** [review]
Does this also work for twitter an dothe rsocial meida pages? Which do we want to support?

File: web/src/modules/common/containers/Helmet.js (hunk 1 above)
Comment (klinzo) [reply web/src/modules/common/containers/Helmet.js:39]: I'll add a little more info here.

**Comment by klinzo** [comment]
> Does this also work for twitter and other social media pages? Which do we want to support?

According to this post it works for facebook, twitter and linkedin at least. I think we just have to test afterwards whether this is working for the platforms we need as I don't see a way to test this without deploying first.

**Comment by maxammann** [comment]
Oke cool, did not read the post :) that is very fine

**Comment by klinzo** [comment]
> Oke cool, did not read the post :) that is very fine

Sorry, forgot to include the post :facepalm: https://ahrefs.com/blog/open-graph-meta-tags/

**Review by Taggotty — APPROVED** [review]

**Pull request merged**